tests/integration/test_auth.py

```diff
@@ -55,76 +47,12 @@ async def test_deploy_minimal_kyuubi_setup(
             APP_NAME,
             charm_versions.integration_hub.application_name,
             charm_versions.s3.application_name,
+            charm_versions.auth_db.application_name,
         ],
         idle_period=20,
         status="active",
     )
 
-    # Assert that all charms that were deployed as part of minimal setup are in correct states.
-    assert check_status(ops_test.model.applications[APP_NAME], Status.ACTIVE.value)
-    assert (
-        ops_test.model.applications[charm_versions.integration_hub.application_name].status
-        == "active"
-    )
-    assert ops_test.model.applications[charm_versions.s3.application_name].status == "active"
-
-
-@pytest.mark.abort_on_fail
-async def test_sql_queries_no_authentication(ops_test):
```

Comment:
I think these tests are still relevant here.

Reply:
I'd still prefer to keep this here since this test here, while in `test_charm` we can test just the happy path (since we have different modules that intensively test each specific aspect of the charm. However, this is not a big issue, and I leave this upto you.